### DISCUSSION TITLE
Settings: Role App Access Callback URL

### DISCUSSION THREAD

**Discussion by JonathanSchndr** [directus/directus#15708] [Feature Requests]
Currently a role can be prevented from logging into the app. This is very useful if you program your own frontend and you want to prevent a user from using the default backend with his permissions.

![Bildschirmfoto 2022-09-22 um 15 56 43](https://user-images.githubusercontent.com/922406/191773979-19115211-cc16-4e9c-841e-3dff18bb086f.png)

Nevertheless, the user can of course authenticate himself or reset the password. Here it would be good to define a callback URL in the settings, which forwards the user after successful login or password reset - for example to his own frontend incl. JWT token.

If a user tries to log in without app access, an error message is displayed. This would then also be covered.

![Bildschirmfoto 2022-09-22 um 16 32 06](https://user-images.githubusercontent.com/922406/191775309-0e32d4de-bf8d-4e1d-b84d-265761d91174.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
